build.sbt

```diff
@@ -36,21 +35,15 @@ def common: Seq[Setting[_]] = releaseSettings ++ bintraySettings ++ evictionSett
   licenses := Seq(("Apache-2.0", url("http://www.apache.org/licenses/LICENSE-2.0.html"))),
   homepage := Some(url("https://www.lagomframework.com/")),
   sonatypeProfileName := "com.lightbend",
-  headers := headers.value ++ Map(
-     "scala" -> (
-       HeaderPattern.cStyleBlockComment,
-       """|/*
-          | * Copyright (C) 2016-2018 Lightbend Inc. <https://www.lightbend.com>
-          | */
-          |""".stripMargin
-     ),
-     "java" -> (
-       HeaderPattern.cStyleBlockComment,
-       """|/*
-          | * Copyright (C) 2016-2018 Lightbend Inc. <https://www.lightbend.com>
-          | */
-          |""".stripMargin
-     )
+  headerLicense := Some(HeaderLicense.Custom(
+    """|/*
+       | * Copyright (C) 2016-2018 Lightbend Inc. <https://www.lightbend.com>
+       | */
+       |""".stripMargin
+  )),
+  headerMappings := headerMappings.value ++ Map(
+    HeaderFileType.scala -> HeaderCommentStyle.cStyleBlockComment,
+    HeaderFileType.java -> HeaderCommentStyle.cStyleBlockComment
```

Comment:
Code-Duplication Gods will make you pay for this.

Reply:
How dare you, _developer_, remove some duplication! 😂

Reply:
So you want `Seq(HeaderFileType.scala, HeaderFileType.java).map(_ -> HeaderCommentStyle.cStyleBlockComment)`? 😅

Reply:
Thy wills be done, O great Code-Dupe Gods.

Reply:
I was actually praising how you deduplicated the "Big Header String"-thing. :-D

Reply:
Ha! Well, now it's even less duplicated. Good karma coming my way any day now.

Reply:
...Ojalá que sí... 😄